packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-attach/app-attach.html

```diff
@@ -0,0 +1,200 @@
+<!--App attach object storage-->
+<div class="mb-2">
+    <!--Description-->
+    <span>
+        <span
+            class="mb-0"
+            data-translate="pci_app_add_attach_container_description"
+        ></span>
+        <a
+            data-ng-href="{{:: $ctrl.getStorageInfoLink() }}"
+            target="_blank"
+            rel="noopener"
+        >
+            <span
+                data-translate="pci_app_add_attach_container_description_link"
+            ></span>
+            <span class="oui-icon oui-icon-target" aria-hidden="true"></span>
+        </a>
+    </span>
+</div>
+
+<!--App storage labels-->
+<div class="mb-4">
+    <div class="oui-box oui-box_light pl-0 pb-2 border-0">
+        <!--Manage labels-->
+        <div class="oui-inline-adder mb-2">
+            <div
+                class="oui-inline-adder__form"
+                data-ng-repeat="volume in $ctrl.volumes track by $index"
+            >
+                <fieldset class="oui-inline-adder__fieldset">
```

Comment:
Same here please, maybe add `legend` for the fieldsets

Reply:
I've check all the `<fieldset class="oui-inline-adder__fieldset">` none had a legend.

Reply:
Yes, it is not done for the moment, but it should ;)

Reply:
I understand, but

- I don't know what to put
- I don't know the look of it after.

So, as I'm the first, I will not do it without an example